packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -948,6 +987,36 @@ describe('Checkout', () => {
             });
         });
 
+        describe('loadBasket ::', () => {
+            describe('when `getBasket` request fails', () => {
+                const wrapper = mount(VueCheckout, {
+                    store: createStore(defaultState, { ...defaultActions, getBasket: jest.fn(async () => Promise.reject()) }),
+                    i18n,
+                    localVue,
+                    propsData
+                });
+
+                it('should emit failure event', async () => {
+                    expect(wrapper.emitted(EventNames.CheckoutBasketGetSuccess)).toBeUndefined();
+                    expect(wrapper.emitted(EventNames.CheckoutBasketGetFailure).length).toBe(1);
+                });
+            });
+
+            describe('when `getBasket` request succeeds', () => {
+                const wrapper = mount(VueCheckout, {
+                    store: createStore(),
+                    i18n,
+                    localVue,
+                    propsData
+                });
+
+                it('should emit success event', async () => {
+                    expect(wrapper.emitted(EventNames.CheckoutBasketGetSuccess).length).toBe(1);
+                    expect(wrapper.emitted(EventNames.CheckoutBasketGetFailure)).toBeUndefined();
```

Comment:
Probably worth having this a separate `IT`:

`it('should not emit a 'CheckoutBasketGetFailure' when basket is successful')`

Reply:
In fact i dont think you need this one as the one above looks to cover the failure. So this test really only needs the expect on the line above. Then the test above only needs the failure one.

Reply:
Fixed